doc/admin-guide/security/index.en.rst

```diff
@@ -281,6 +281,11 @@ Authority Information Access field of the signed certificate. For example::
                 OCSP - URI:http://ocsp.digicert.com
                 CA Issuers - URI:http://cacerts.digicert.com/DigiCertSHA2SecureServerCA.crt
 
+Traffic Server can also use prefetched OCSP stapling responses if ssl_ocsp_name parameter
+is used in :file:`ssl_multicert.config`. Take into account that when using prefetched
+OCSP stapling responses traffic server will not refresh them and it should be done
+externally.
+
```

Comment:
Could we give an example of how to do this? Something like a curl command to show what the file content is supposed to contain (body only? headers plus body?) would be helpful.

Reply:
Indeed, for my tests I'm using: `openssl ocsp -issuer /etc/acmecerts/rsa.chain.crt -cert /etc/acmecerts/rsa.crt -host ocsp.int-x3.letsencrypt.org:80 -header "Host=ocsp.int-x3.letsencrypt.org" -respout /etc/acmecerts/rsa.ocsp`.
That can definitely be added in the documentation :)